### PR TITLE
UI Modification: When New Robot

### PR DESCRIPTION
**Pull request by AllForNothing**
#8083
image
Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts [hunk 1]
```diff
@@ -118,13 +121,14 @@ export class AddRobotComponent implements OnInit, OnDestroy {
...
-      this.robot.access.isPushOrPullImage = false;
-    } else {
+    if ( this.imagePermissionPush && this.imagePermissionPull) {
       this.robot.access.isPullImage = false;
       this.robot.access.isPushOrPullImage = true;
+    } else {
+      this.robot.access.isPullImage = true;
+      this.robot.access.isPushOrPullImage = false;
     }
+    console.info(this.robot.access);
```
Comment (zhoumeina) [inline src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: remove this console.

File: src/portal/src/i18n/lang/en-us-lang.json [hunk 2]
```diff
@@ -322,7 +322,8 @@
         "CREATED_SUCCESS": "Created '{{param}}' successfully.",
         "COPY_SUCCESS": "Copy token successfully of '{{param}}'",
         "DELETION_TITLE": "Confirm removal of robot accounts",
-        "DELETION_SUMMARY": "Do you want to delete robot accounts {{param}}?"
+        "DELETION_SUMMARY": "Do you want to delete robot accounts {{param}}?",
+        "PULL_IS_MUST" : "Pull permission is checked by default and can not modify"
```
Comment (zhoumeina) [inline src/portal/src/i18n/lang/en-us-lang.json]: can not be modified.

File: src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts]: removed

File: src/portal/src/i18n/lang/en-us-lang.json (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/i18n/lang/en-us-lang.json]: modified

**Pull request merged**